federation-js/src/composition/validate/preComposition/__tests__/keyFieldsMissingExternal.test.ts

```diff
@@ -133,6 +141,12 @@ describe('keyFieldsMissingExternal', () => {
       Array [
         Object {
           "code": "KEY_FIELDS_MISSING_EXTERNAL",
+          "locations": Array [
+            Object {
+              "column": 35,
+              "line": 1,
+            },
```

Comment:
This seems odd

Reply:
I can't figure out why this doesn't give a reasonable location. The node seems right. I have switched it to the parent type.